src/lib/select/select.ts

```diff
@@ -360,6 +360,9 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     this._multiple = coerceBooleanProperty(value);
   }
 
+  /** Whether to center the active option over the trigger. */
+  @Input () disableOptionCentering: boolean = false;
```

Comment:
Can you use `coerceBooleanProperty()` for this? (see above for examples)

Reply:
Thanks for the suggestion! I've implemented it and it seems to make it easier for the user to customize now.